tests/resources/example_project/greeter.py

```diff
@@ -13,4 +13,3 @@
     greeting = [args.greeting, args.name]
     if args.excitement is not None:
         greeting.append("!" * args.excitement)
-    print(" ".join(greeting))
```

Comment:
Let's keep this line.

Reply:
Should I add an `# pylint: disable-next=print-function` also?

Reply:
By the moment I keep it avoiding the print warning